chain/chain/src/tests/simple_chain.rs

```diff
@@ -36,6 +36,7 @@ fn build_chain() {
     let _mock_clock_guard = MockClockGuard::default();
     for i in 0..5 {
         Clock::add_utc(chrono::Utc.ymd(2020, 10, 1).and_hms_milli(0, 0, 3, 444 + i));
+        Clock::add_utc(chrono::Utc.ymd(2020, 10, 1).and_hms_milli(0, 0, 3, 444 + i));
```

Comment:
can you comment why are we adding it two times?

Reply:
is the second call here due to 'validate_header' call?

Reply:
yes, the validate header consumes 4 more entries from the clock in this test scenario.